### PR TITLE
Brand refresh updates/structure

### PR DESCRIPTION
**Pull request by RobTobias123**
Updated images with latest brand and as webp format residing in images folder. Updated text to fit the latest way things work.

### CODE REVIEW

File: docs/foundations/structure.md [hunk 1]
```diff
@@ -122,26 +122,25 @@ The cards that adapt the most are `section` and `support with image`
...
 ### `<ns-landmark>`
 
-Lots of responsive decisions are contained within each `ns-landmark` for example, the `summit` variant there is no `card` at `squash-court` and `tennis-court` and at `basketball-court` the card overlaps the image and sticks to 
-the left. At `hockey-pitch` and above the card floats over the image.
+Lots of responsive decisions are contained within each `ns-landmark` for example, the `summit` variant there is no `card` at `squash-court` and `tennis-court` and at `basketball-court` the card overlaps the image. At `hockey-pitch` and above the card sticks to the left in a 60%/40% layout.
 
 ### `<ns-lockup>`
 
-All three of our lockups behave uniquely responsive.
+The ns-lockup behaves responsively too, adjusting it's overlaps most appropriately for the given viewport.
```
Comment (PtitBen) [inline docs/foundations/structure.md:137]:
```suggestion
The ns-lockup behaves responsively too, adjusting its overlaps most appropriately for the given viewport.
```

File: docs/foundations/structure.md [hunk 2]
```diff
@@ -69,41 +69,41 @@ This piece of our structure provides a container for a single topic or task. Thi
 
 ## 3. Layout
 
-Here I refer to the layout of our components into three widths, Wide, Norm and Slim.
+The general layout of our components fit within a content area in between margins that are generally and equally 5% of the total viewport width, up to 1,600 px wide. When that happens, the main content area remains fixed at that size, but the left and right margins then expand equally to fill the rest of the horizontal space as the viewport width increases over 1,600 px.
 
-The adding of a class to render different types of layout allows us to implement a modern and dynamic aspect of our creative principles.
+It allows our layouts to respond to different devices dynamically and be optimal for comfortable scanning.
```
Comment (PtitBen) [inline docs/foundations/structure.md:74]:
```suggestion
It allows our layouts to respond to different devices dynamically and be optimal for comfortable scanning and content consumption.
```

File: docs/foundations/structure.md [hunk 3]
```diff
@@ -69,41 +69,41 @@ This piece of our structure provides a container for a single topic or task. Thi
 
 ## 3. Layout
 
-Here I refer to the layout of our components into three widths, Wide, Norm and Slim.
+The general layout of our components fit within a content area in between margins that are generally and equally 5% of the total viewport width, up to 1,600 px wide. When that happens, the main content area remains fixed at that size, but the left and right margins then expand equally to fill the rest of the horizontal space as the viewport width increases over 1,600 px.
```
Comment (PtitBen) [inline docs/foundations/structure.md:72]:
```suggestion
The general layout of our components fits within a content area in between margins that are generally and equally 5% of the total viewport width, up to 1,600 px wide. When that happens, the main content area remains fixed at that size, but the left and right margins then expand equally to fill the rest of the horizontal space as the viewport width increases over 1,600 px.
```

**Comment by RobTobias123** [comment]
Committed suggestions - thanks Ben.

**Pull request merged**